app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    source, target = parts[1], parts[2]
+    if len(parts) != 3 or parts[0] != "mv" or source == target:
+        return
+
+    if os.path.dirname(target) != "":
```

Comment:
You can simply check it with `if target`.

Reply:
i don't understand

Reply:
first you need  make this condition
![Screenshot from 2023-09-22 14-41-55](https://github.com/mate-academy/py-move-file/assets/122517203/2deee661-0c0b-46c8-bf83-204ce89eabe5)